packages/commonwealth/client/scripts/views/components/SocialAccount.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { CWIcon } from './component_kit/cw_icons/cw_icon';
+import type { IconName } from './component_kit/cw_icons/cw_icon_lookup';
+
+type SocialAccountProps = {
+  iconName: IconName;
+  link: string;
+};
+
+export const SocialAccount = (props: SocialAccountProps) => {
```

Comment:
destructure the `props` in-line

Reply:
@lzach83 good catch. Just pushed the change.